network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -1192,13 +1209,24 @@ class BuyersGuidePage(RoutablePageMixin, FoundationMetadataPageMixin, Page):
     def get_banner(self):
         return self.hero_image
 
+    def get_excluded_categories(self):
+        return [
+            {"name": category.category.name} for category in self.excluded_categories.all()
```

Comment:
just to avoid confusion, let's make this use the same name as later in the code:
```suggestion
            {"name": excats.category.name} for excats in self.excluded_categories.all()
```